diff --git a/cmake/FindDependencies.cmake b/cmake/FindDependencies.cmake
index 3c0e259bb..0b0689b2b 100644
--- a/cmake/FindDependencies.cmake
+++ b/cmake/FindDependencies.cmake
@@ -342,57 +342,6 @@ if(ONNX_ENABLED)
         set(onnxruntime_LIBRARY_DIR_HINTS ${onnxruntime_BINARY_DIR}/lib CACHE PATH "ONNX Runtime library directory hints")
         find_package(onnxruntime ${COLMAP_FIND_TYPE})
 
-        install(DIRECTORY "${onnxruntime_BINARY_DIR}/include/" TYPE INCLUDE)
-        if(IS_WINDOWS)
-            # On Windows, selectively install Libs to lib/. Always install core Libs.
-            # For not supporting TensorRT/ROCM/etc. as a runtime, so not installing it intentionally.
-            install(FILES
-                "${onnxruntime_LIB_DIR}/onnxruntime.lib"
-                "${onnxruntime_LIB_DIR}/onnxruntime_providers_shared.lib"
-                TYPE LIB)
-            # Only install CUDA provider Lib if CUDA is enabled.
-            if(CUDA_ENABLED)
-                install(FILES
-                    "${onnxruntime_LIB_DIR}/onnxruntime_providers_cuda.lib"
-                    TYPE LIB)
-            endif()
-            # On Windows, selectively install DLLs to bin/. Always install core DLLs.
-            # For not supporting TensorRT/ROCM/etc. as a runtime, so not installing it intentionally.
-            install(FILES
-                "${onnxruntime_LIB_DIR}/onnxruntime.dll"
-                "${onnxruntime_LIB_DIR}/onnxruntime_providers_shared.dll"
-                TYPE BIN)
-            # Only install CUDA provider DLL if CUDA is enabled.
-            if(CUDA_ENABLED)
-                install(FILES
-                    "${onnxruntime_LIB_DIR}/onnxruntime_providers_cuda.dll"
-                    TYPE BIN)
-            endif()
-        else()
-            # On Linux/macOS, selectively install library files. Always install core libraries.
-            # Not supporting TensorRT/ROCM/etc. as a runtime, so not installing them.
-            if(IS_MACOS)
-                file(GLOB onnxruntime_CORE_LIBS
-                    "${onnxruntime_LIB_DIR}/libonnxruntime.dylib"
-                    "${onnxruntime_LIB_DIR}/libonnxruntime.*.dylib"
-                    "${onnxruntime_LIB_DIR}/libonnxruntime_providers_shared.dylib")
-                install(FILES ${onnxruntime_CORE_LIBS} TYPE LIB)
-            else()
-                file(GLOB onnxruntime_CORE_LIBS
-                    "${onnxruntime_LIB_DIR}/libonnxruntime.so*"
-                    "${onnxruntime_LIB_DIR}/libonnxruntime_providers_shared.so*")
-                install(FILES ${onnxruntime_CORE_LIBS} TYPE LIB)
-                # Only install CUDA provider if CUDA is enabled.
-                if(CUDA_ENABLED)
-                    file(GLOB onnxruntime_CUDA_LIBS
-                        "${onnxruntime_LIB_DIR}/libonnxruntime_providers_cuda.so*")
-                    install(FILES ${onnxruntime_CUDA_LIBS} TYPE LIB)
-                endif()
-            endif()
-        endif()
-        if(EXISTS "${onnxruntime_BINARY_DIR}/share")
-            install(DIRECTORY "${onnxruntime_BINARY_DIR}/share/" TYPE DATA)
-        endif()
 
         message(STATUS "Configuring onnxruntime... done")
     else()
